### ISSUE TITLE
[Idea] Support for Tails mod

### ISSUE THREAD

**Issue by Vexatos** [asiekierka/Statues#5]
Just leaving this here, I might get to add support for this mods some time.

**Comment by asiekierka** [comment]
Also MorePlayerModels. I am familiar with its APIs somewhat, so...

**Comment by Vexatos** [comment]
Good idea, More Player Models is awesome! Unfortunately, that would need a whole new GUI tab to customise as you cannot get the model etc as easily as you get it from Tails.
